src/features/rewards/mainToggle/index.tsx

```diff
@@ -22,11 +22,20 @@ export interface Props {
   onToggle: () => void
   id?: string
   testId?: string
+  onTOSClick: () => void
```

Comment:
in other places you set this two as optional, can you please set it here as well?

Reply:
done